client/tests/get_status_test.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+import featureform as ff
+from featureform.resources import TrainingSet, Feature, ResourceColumnMapping, ResourceStatus
+from featureform.serving import ServingClient
+
+from datetime import timedelta
+
+from .proto import metadata_pb2_grpc, metadata_pb2
+
+def resource_with_status(resource, status):
+    resource.status = status
+    return resource
+
+expected_list = [("PENDING", ff.ResourceStatus.PENDING), ("READY", ff.ResourceStatus.READY), ("CREATED", ff.ResourceStatus.CREATED), ("FAILED", ff.ResourceStatus.FAILED)]
+
+@pytest.mark.paramaterize(input, expected, expected_list)
```

Comment:
Should check the formatting for this. I think its like @pytest.mark.paramaterize("input,expected", expected_list)